utility/utils.py

```diff
@@ -382,8 +382,11 @@ def configure_logger(test_name, run_dir, level=logging.INFO):
     )
     _handler.setFormatter(formatter)
     _root.addHandler(_handler)
-
-    url_base = "http://magna002.ceph.redhat.com/cephci-jenkins"
+    url_base = (
```

Comment:
This looks good, thanks for fixing the links.

Would it make sense to have a variable having the value `http://magna002.ceph.redhat.com/cephci-jenkins`? For example `file_server_url` or `magna_url`.